vmdb/app/models/mixins/miq_web_server_worker_mixin.rb

```diff
@@ -54,7 +54,7 @@ def self.build_command_line(*params)
     end
 
     def self.all_ports_in_use
-      self.server_scope.all.collect { |w| w.port unless w.is_stopped? && !MiqProcess.is_worker?(w.pid)}.compact
+      server_scope.select { |w| MiqProcess.is_worker?(w.pid) || !w.is_stopped? }.collect(&:port)
```

Comment:
My theory behind flipping it was that 1. it reads more naturally [to me] to check its worker-ness before we check whether it's stopped, and 2. although I did suspect this was moving the expensive arm to the left, my intuition was that this wouldn't be hot enough for it to matter.

Happy to flip it back (or swap the `select` for a `reject`, if the negative version makes more sense?)